doc/content/examples/index.md

```diff
@@ -24,5 +24,7 @@ A list of example simulations using MASTODON is provided below. These examples i
 - [Example 12a](examples/example12a.md): Fluid-structure interaction analysis of a tank subjected to uni-directional harmonic acceleration.
 - [Example 12b](examples/example12b.md): Fluid-structure interaction analysis of a tank subjected to uni-directional earthquake acceleration.
 - [Example 12c](examples/example12c.md): Experimental validation of fluid-structure interaction simulations.
+- [Example 13](examples/example13.md): Soil-Structure Interaction and Fluid Structure Interaction in nuclear power plant structure.
+- [Example 14](examples/example14.md): Seismic analysis of a base-isolated nuclear power plant building.
 
```

Comment:
Did you mean to add the ex 13 text here?

Reply:
Yeah I just did a rebase with your branch so that there wont be a merge conflict later.